pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -498,26 +511,30 @@ func (l *LibvirtDomainManager) preStartHook(vmi *v1.VirtualMachineInstance, doma
 	if err := emptydisk.NewEmptyDiskCreator().CreateTemporaryDisks(vmi); err != nil {
 		return domain, fmt.Errorf("creating empty disks failed: %v", err)
 	}
+
+	// If we are a migration target, we care about image size, not contents
+	generateEmptyIsos := isMigrationTarget
```

Comment:
I know it may seem superfluous, but I did that for semantics purposes.
We want preStartHook to have 2 modes of execution, new VMI or migration target.
But then when calling the disk creation functions, we want to tell them whether to create a populated iso or an empty one.
Both booleans happen to equal each other, but they have separate meanings.
Anyway, sorry, lots of talking for a simple bool, let me know if you feel strongly about it and I'll merge them.

Reply:
not strongly, but I don't understand your reasoning as well... :)
There is nothing in this function that uses  `isMigrationTarget`. 
Also, it is pretty clear (to me) that there are two modes when you call `preStartHook(vmi, domain, generateEmptyIsos=true/false)`

Reply:
Fixed :)